pymc/tests/variational/test_inference.py

```diff
@@ -149,14 +149,13 @@ def fit_kwargs(inference, use_minibatch):
         inference.approx.scale_cost_to_minibatch = False
     else:
         key = "full"
+    if (type(inference), key) in {(SVGD, "mini"), (ASVGD, "mini")}:
+        pytest.skip("Not Implemented Inderence")
```

Comment:
Should it be an `xfail`?
```suggestion
        pytest.skip("Not Implemented Inference")
```

Reply:
This is a part of a bigger issue, all references to NotImplementedInference should be xfail

Reply:
Basically, NotImplementedInference is something that was broken because of v3-v4 update and we know about it